Initial commit of labs project moved from Sample Pages repo (#46)
skuid/skuid-labs: experiments/snippets/realTimeOppDashboard/README.md

> # Real Time Opportunity Dashboard

Reviewer: As is, this page *does not work*. 

I did get it working by using an action sequence, but if we want this to be more universal, and since we're already in snippet land, we have two options:

- Change `poll for updates` into an In-Line (Snippet) snippet type. Create an event-triggered action sequence that then runs that snippet on Skuid Page: Rendered
- Wrapping the snippet with our recommended method of waiting for page rendering as of 11.1.0:

```
$(document.body).one('pageload',function() { 
// Will fire after Skuid is finished fully loading the page
// Code goes here
})
```

Author: I added the code to the snippet.  It seems to work now.

> Skuid pages generally only query the database on user request.  The user reloads the page, or takes some action on the page to trigger a query.  But customers often ask whether Skuid can create a page where new data is "pushed".  

Reviewer: ```suggestion
Skuid pages generally only query the database on user request.  The user reloads the page or takes some action on the page to trigger a query.  But customers often ask whether Skuid can create a page where new data is constantly updated—also known as "polling."  
```

This sentence says the opposite of what we mean. Also a comma.

> This page extends Skuid with a simple snippet that polls a "updates indicator" model periodically.  If this model returns data - the user can be warned that there are updates and can update the page. 

Reviewer: ```suggestion
This page uses a simple snippet that polls an "updates indicator" model periodically.  If this model returns data, the user is warned that there is new data, and they can update the page. 
```

> It is not wise to simply continuously poll the primary model, as that would freeze the components bound to that model whenever the queries are occuring,  and that cause problems if there were unsaved chagnes. 

Reviewer: Minor style tweaks, a lot of adverbs here. Also spelling, verb tense, and whitespace.
```suggestion
It is not wise to just continuously poll the primary model, as that would freeze the components bound to that model whenever the queries occur. These frozen components would then cause problems for any unsaved changes. 
```

Reviewer: @Rnhatch

Author: I'm going to rewrite sentence cuz I'm not happy with either option.
